test/lib/game/round_test.exs

```diff
@@ -0,0 +1,17 @@
+defmodule RoundTest do
+  use ExUnit.Case, async: true
+
+  test "start_link/1 takes users and sets state" do
+    users = ["Tim", "Eli"]
+    {:ok, pid} = Reactor.Game.Round.start_link(users)
+    assert is_pid(pid)
```

Comment:
This test doesn't really test any of your code – you're never interrogating that the users have been set. You're mainly testing `start_link` returning a pid here which you can guarantee.